state/internal_test.go

```diff
@@ -76,3 +86,25 @@ func (s *internalStateSuite) TearDownTest(c *gc.C) {
 	s.BaseSuite.TearDownTest(c)
 	s.MgoSuite.TearDownTest(c)
 }
+
+type internalStatePolicy struct{}
```

Comment:
The other methods will be used. NotImplemented errors are treated specially by the policy-related code; if such an error is encountered, it is as if there is no policy at all.